indico/modules/events/roles/util.py

```diff
@@ -25,3 +32,31 @@ def serialize_event_role(role, legacy=True):
             'name': role.name,
             'identifier': 'EventRole:{}'.format(role.id),
         }
+
+
+def import_members_from_csv(f):
+    """Import members from a CSV file into an event role."""
+    reader = csv.reader(f.read().splitlines())
+    members_data = set()
+
+    for num_row, row in enumerate(reader, 1):
+        try:
+            email = [to_unicode(value).strip() for value in row]
+        except ValueError:
+            raise UserValueError(_('Row {}: malformed CSV data').format(num_row))
+        try:
+            email = email[0].lower()
+        except IndexError:
+            continue
+
+        if email and not validate_email(email):
+            raise UserValueError(_("Row {row}: invalid email address: {email}").format(row=num_row, email=email))
+        if email in members_data:
+            raise UserValueError(_('Row {}: email address is not unique').format(num_row))
+        members_data.add(email)
+
+    registered_users = set(User.query.filter(~User.is_deleted, User.all_emails.in_(list(members_data))))
+    registered_users_email = set(user.email for user in list(registered_users))
```

Comment:
you can iterate a set directly, without converting it to a list first, and python 2.7+ has set comprehensions so no need to use `set(...)` - you can use `{...}` instead